recipes/platetectonictools/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "PlateTectonicTools" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0]|lower }}/{{ name|lower }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3935e7846f7c16040a04ca689b10be8378afe6573dc8c4eaae0cf6b75f33c07a
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

This looks like it's pure Python and can be noarch:
```suggestion
build:
  number: 0
  script: "{{ PYTHON }} -m pip install . -vv"
  noarch: python
```